joomla/joomla-cms
# Fix subform chosen

Pull Request for Issue # .

### Summary of Changes
When adding a new row in a subform that contains one or more selects that should be turned into 'chosen', chosen initialization will be run on the new row's elements.

This was originally part of #12542 but separating it to keep PRs small and simple. 

### Testing Instructions
Add new rows to any subform that contains 'chosen'. Before this change, they should remain as regular selects. After, they will be proper chosens. 

### Documentation Changes Required
No
